proxy/envoy/route.go

```diff
@@ -216,6 +234,15 @@ func buildCluster(address, name string, timeout *duration.Duration) *Cluster {
 	}
 }
 
+func buildDecorator(config model.Config) *Decorator {
+	if config.ConfigMeta.Name != "" {
+		return &Decorator{
+			Operation: config.ConfigMeta.Name,
```

Comment:
Please use `config.Key()` or include the namespace. Names are not unique.

Reply:
The intention is not to be unique but to enable users to specify a business relevant name that can be used in the tracing operation/span name.